EventDisplay/src/EventDisplayFrame.cc

```diff
@@ -48,6 +48,7 @@ namespace mu2e_eventdisplay
 EventDisplayFrame::EventDisplayFrame(const TGWindow* p, UInt_t w, UInt_t h, fhicl::ParameterSet const &pset) :
   TGMainFrame(p, w, h),
   _wideband(pset.get<bool>("wideband",false)),
+  _extracted(pset.get<bool>("extracted",false)),
```

Comment:
For a future PR.  Please upgrade to validated fcl.

Reply:
What is "validated fcl"?

Reply:
It's a change to your module code - the fcl itself is unchanged.  In your module you provide a description of what parameters must be present in a valid parameter set and what parameters may optionally be present in a valid parmaeter set.  When fhicl parses the parameter set it will recognize missing parameters.  It will also give an error if it finds parameters that do not belong or are of the wrong type.  There is some documentation at https://mu2ewiki.fnal.gov/wiki/FclIntro#The_C.2B.2B_API_.28Validated_FHiCL.29  and https://cdcvs.fnal.gov/redmine/projects/fhicl-cpp/wiki/Configuration_validation_and_fhiclcpp_types

Reply:
Ok. Thank you. I have used this style at other modules before.